src/helix-ui/styles/components/radios.less

```diff
@@ -0,0 +1,255 @@
+@import (reference) 'mixins';
+@import (reference) 'vars';
+
+// BASE STYLES
+hx-radio {
+  border-radius: 1rem;
+  display: inline-block;
+  height: 1rem;
+  vertical-align: baseline;
+  width: 1rem;
+}
+
+// LAYOUT GEOMETRY
+hx-radio-control {
+  -ms-grid-column-align: start;
+  -ms-grid-columns: auto 1fr;
+  -ms-grid-rows: auto auto;
+  display: -ms-grid;
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto auto;
+  grid-template-areas:
+    'ctrl lbl'
+    '.    help';
+  justify-items: start;
+
+  > * {
+    margin: 0;
+  }
+
+  // facade + text
+  > label {
+    // grid child
+    -ms-grid-column: 2;
+    -ms-grid-row: 1;
+    grid-area: lbl;
+
+    // grid parent
+    -ms-grid-column: auto 1fr;
+    -ms-grid-row: auto;
+    display: -ms-grid;
+    display: grid;
+    grid-template-areas: 'facade txt';
+    grid-template-columns: auto 1fr;
+    grid-template-rows: auto;
+    grid-gap: 0.5rem;
+    padding: 0;
+
+    > hx-radio {
+      -ms-grid-column: 1;
+      -ms-grid-row: 1;
+      grid-area: facade;
+      display: none;
+      margin-top: 0.25rem;
+    }
+  }
+
+  > p {
+    -ms-grid-column: 2;
+    -ms-grid-row: 2;
+    grid-area: help;
+    margin: 0;
+  }
+
+  > input[type="radio"] {
+    -ms-grid-column: 1;
+    -ms-grid-row: 1;
+    border-radius: 1rem;
+    grid-area: ctrl;
+    height: 1rem;
+    margin: 0.25rem 0.5rem 0 0;
+    width: 1rem;
+
+    &::-ms-check {
+      box-sizing: border-box;
+      display: block;
```

Comment:
(not a deal breaker) Do we need this `display: block;` definition?

Maybe set a background color with `!important` and remove this line to see if it fills as expected.

Reply:
Removed `display:block`.  It didn't appear to make a difference.